docs/getting_started/faq.md

```diff
@@ -36,31 +35,26 @@ The following are answers to some frequently asked questions (FAQs) about {{site
 - [How do I manage AI at the Edge with models vs AI on the Cloud?](#ai_cloud)
 
 ## Is there a way to create a self-contained environment for development purposes?
-
 {: #one_click}
 
-You can install the open source management hub (without the {{site.data.keyword.ieam}} management console) with the "all-in-one" installer for developers. The all-in-one installer creates a complete but minimal management hub, not suitable for production use. It also configures an example edge node. This tool enables open source component developers to get started quickly without the time it takes to configure a complete production {{site.data.keyword.ieam}} management hub. For information about the all-in-one installer, see [Open Horizon - Devops ](https://github.com/open-horizon/devops/tree/master/mgmt-hub){:target="\_blank"}{: .externalLink}.
+You can install the open source management hub (without the {{site.data.keyword.ieam}} management console) with the "all-in-one" installer for developers. The all-in-one installer creates a complete but minimal management hub, not suitable for production use. It also configures an example edge node. This tool enables open source component developers to get started quickly without the time it takes to configure a complete production {{site.data.keyword.ieam}} management hub. For information about the all-in-one installer, see [Open Horizon - Devops ](https://github.com/open-horizon/devops/tree/master/mgmt-hub){:target="_blank"}{: .externalLink}.
 
 ## Is {{site.data.keyword.ieam}} software open-sourced?
-
 {: #open_sourced}
 
-{{site.data.keyword.ieam}} is an IBM product. But its core components heavily use the [Open Horizon - EdgeX Project Group ](https://wiki.edgexfoundry.org/display/FA/Open+Horizon+-+EdgeX+Project+Group){:target="\_blank"}{: .externalLink} open source project. Many sample and example programs that are available in the {{site.data.keyword.horizon}} project work with {{site.data.keyword.ieam}}. For more information about the project, see [Open Horizon - EdgeX Project Group ](https://wiki.edgexfoundry.org/display/FA/Open+Horizon+-+EdgeX+Project+Group){:target="\_blank"}{: .externalLink}.
+{{site.data.keyword.ieam}} is an IBM product. But its core components heavily use the [Open Horizon - EdgeX Project Group ](https://wiki.edgexfoundry.org/display/FA/Open+Horizon+-+EdgeX+Project+Group){:target="_blank"}{: .externalLink} open source project. Many sample and example programs that are available in the {{site.data.keyword.horizon}} project work with {{site.data.keyword.ieam}}. For more information about the project, see [Open Horizon - EdgeX Project Group ](https://wiki.edgexfoundry.org/display/FA/Open+Horizon+-+EdgeX+Project+Group){:target="_blank"}{: .externalLink}.
```

Comment:
The following link is legacy from before Open Horizon was a standalone project:

[Open Horizon - EdgeX Project Group ](https://wiki.edgexfoundry.org/display/FA/Open+Horizon+-+EdgeX+Project+Group)

Suggest changing it to the official project page at:

[Open Horizon, an LF Edge project](https://www.lfedge.org/projects/openhorizon/)

Reply:
I fixed the LFEdge Open Horizon URL in the FAQ in https://github.com/open-horizon/open-horizon.github.io/pull/290/commits/193d1a850b5021a46b6a1ac6a0816af0886fa9e8

Reply:
Thanks for the review.